appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -103,6 +103,19 @@
 		}
 	}
 
+	private void TeasingSonarQube()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "TeasingSonarQube" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS00100)